src/config/keys.ts

```diff
@@ -103,6 +103,10 @@ export const buildGetLikedItemsKey = (id: UUID) => [
 ];
 export const buildGetLikeCountKey = (id: UUID) => [ITEMS_KEY, id, 'likeCount'];
 
+export const VALIDATION_REVIEW_KEY = 'itemValidationReviews';
+export const ALL_STATUS_KEY = 'allValidationStatus';
+export const buildValidationStatusKey = (id: UUID) => [ITEMS_KEY, id, 'validationStatus'];
```

Comment:
Is it supposed to contain only one status or multiple ones?

Reply:
It contains multiple, but just related to given item